.github/workflows/integrate.yaml

```diff
@@ -7,6 +7,7 @@ on:
       - 'plutus-extra.cabal'
     branches:
       - master
+      - staging
```

Comment:
This can be removed, since I don't plan to retain `staging` as a working branch.